### PR TITLE
Update free-programming-books-hi.md

### PR DESCRIPTION
**Pull request by aniket-2002**
## What does this PR do?
a complete guide for java in hindi

### Why is this valuable (or not)?
this is a complete course for java in hindi
### How do we know it's really free?
it is a free website
### For book lists, is it a book? For course lists, is it a course? etc.
it is a complete course
## Checklist:
- [☑️ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ☑️] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ☑️] Include author(s) and platform where appropriate.
- [ ☑️] Put lists in alphabetical order, correct spacing.
- [ ☑️] Add needed indications (PDF, access notes, under construction).
- [ ☑️] Used an informative name for this pull request.

### CODE REVIEW

**Comment by aniket-2002** [comment]
pls merge !

File: books/free-programming-books-hi.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+* [Java](#java)
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="java"></a>Java
```
Comment (shrn01) [inline books/free-programming-books-hi.md:13]:
```suggestion
### Java
```

File: books/free-programming-books-hi.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
+* [Java Notes in Hindi – जावा के हिंदी नोट्स](https://ehindistudy.com/2020/12/05/java-notes-in-hindi/) - eHindiStudy
```
Comment (shrn01) [inline books/free-programming-books-hi.md:15]:
```suggestion
* [Java Notes in Hindi – जावा के हिंदी नोट्स](https://ehindistudy.com/2020/12/05/java-notes-in-hindi) - eHindiStudy
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-hi.md (hunk 2 above)
Comment (aniket-2002) [reply books/free-programming-books-hi.md]: done please check .

**Review by shrn01 — APPROVED** [review]
LGTM

**Comment by eshellman** [comment]
Maybe its all the ads, but this seems like not a book

**Pull request closed without merging**